modules/modules.nix

```
@@ -144,6 +144,7 @@ let
     (loadModule ./services/tahoe-lafs.nix { })
     (loadModule ./services/taskwarrior-sync.nix { condition = hostPlatform.isLinux; })
     (loadModule ./services/udiskie.nix { })
+    (loadModule ./services/urxvtd.nix { })
```

Since this module uses systemd it is Linux only:
```suggestion
    (loadModule ./services/urxvtd.nix { condition = hostPlatform.isLinux; })
```

---

:+1: